slick/src/sphinx/code/LiftedEmbedding.scala

```diff
@@ -266,12 +266,27 @@ object LiftedEmbedding extends App {
         Await.result(result, Duration.Inf)
       }
       {
-        //#delete
+        //#delete1
         val q = coffees.filter(_.supID === 15)
         val action = q.delete
         val affectedRowsCount: Future[Int] = db.run(action)
         val sql = action.statements.head
-        //#delete
+        //#delete1
+        Await.result(affectedRowsCount, Duration.Inf)
+      }
+      {
+        //#delete2
+        //
```

Comment:
This is me being super picky: the extra `//` on the line after `#delete2` gets rendered in the output and looks a bit odd:

![queries slick 3 2 0-snapshot documentation 2016-02-11 16-11-44](https://cloud.githubusercontent.com/assets/102661/12981832/41596612-d0da-11e5-9f08-ddc7bad425cf.png)

The actual example looks good to me!